spec/features/user_logs_in_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.